docker-app/qfieldcloud/subscription/models.py

```diff
@@ -355,16 +355,17 @@ class Package(models.Model):
 
 
 class SubscriptionQuerySet(models.QuerySet):
+    def is_period_active_condition(self):
+        return Q(active_since__lte=V("now")) & (
+            Q(active_until__isnull=True) | Q(active_until__gte=V("now"))
+        )
+
```

Comment:
```suggestion
    @property
    def is_period_active_condition(self):
        return Q(active_since__lte=V("now")) & (
            Q(active_until__isnull=True) | Q(active_until__gte=V("now"))
        )
```

Isn't it easier to use? Or even better, since it is a static content, make it a static property?


```suggestion
    is_period_active_condition = (
        return Q(active_since__lte=V("now")) & (
            Q(active_until__isnull=True) | Q(active_until__gte=V("now"))
        )
    )
```